apps/vr-tests-react-components/src/stories/Rating.stories.tsx

```diff
@@ -0,0 +1,54 @@
+import * as React from 'react';
+import { Steps, StoryWright } from 'storywright';
+import { storiesOf } from '@storybook/react';
+import { CircleRegular, CircleFilled, SquareRegular, SquareFilled } from '@fluentui/react-icons';
+import { Rating } from '@fluentui/react-rating-preview';
+import { TestWrapperDecoratorFixedWidth } from '../utilities/TestWrapperDecorator';
+
+storiesOf('Rating Converged Color', module)
+  .addDecorator(TestWrapperDecoratorFixedWidth)
+  .addDecorator(story => (
+    <StoryWright
+      steps={new Steps()
+        .snapshot('rest', { cropTo: '.testWrapper' })
+        .hover('input')
+        .snapshot('hover', { cropTo: '.testWrapper' })
+        .hover('input'[4])
+        .snapshot('active', { cropTo: '.testWrapper' })
+        .end()}
+    >
+      {story()}
+    </StoryWright>
+  ))
+  .addStory('Neutral Rating with half star', () => <Rating defaultValue={3.5} />, {
+    includeHighContrast: true,
+    includeDarkMode: true,
+  })
+  .addStory('Brand Rating with half star', () => <Rating color="brand" defaultValue={3.5} />, {
+    includeHighContrast: true,
+    includeDarkMode: true,
+  })
+  .addStory('Marigold Rating with half star', () => <Rating color="marigold" defaultValue={3.5} />, {
+    includeHighContrast: true,
+    includeDarkMode: true,
+  });
+
+storiesOf('Rating Converged Size', module)
+  .addDecorator(TestWrapperDecoratorFixedWidth)
```

Comment:
Getting it to crop correctly is always frustrating. Perhaps this will work?
```suggestion
  .addDecorator(TestWrapperDecoratorFixedWidth)
  .addDecorator(story => <StoryWright>{story()}</StoryWright>)
```

Reply:
Hmm, this didn't work. One more thing to try:
```suggestion
  .addDecorator(TestWrapperDecoratorFixedWidth)
  .addDecorator(story => (
    <StoryWright steps={new Steps().snapshot('rest', { cropTo: '.testWrapper' }).end()}>
      {story()}
    </StoryWright>
  ))
```